core/server/api_container/server/startosis_engine/recipe/http_request_recipe.go

```diff
@@ -33,6 +34,8 @@ type HttpRequestRecipe interface {
 	builtin_argument.KurtosisValueType
 
 	Recipe
+
+	RequestType() string
```

Comment:
Should we add a comment? I definitely see myself going through this code in 2 months, being like, this is not used, let's remove it, and then breaking the validation flow. WDYT?

Reply:
haha! I thought of adding one but got lazy. Added it :-)